packages/elements-dev-portal/src/handlers/getNodeContent.ts

```diff
@@ -21,7 +21,10 @@ export const getNodeContent = async ({
   const data = await response.json();
 
   if (!response.ok) {
-    throw new Error(data);
+    throw {
+      status: response.status,
+      data: data,
+    };
```

Comment:
Oh I see where Status above comes from. IMO we should never throw anonymous types, and more generally anything that is not `instanceof Error`. 

Could you make a [custom error class](https://javascript.info/custom-errors) instead?